src/components/GlobalPreferences/Notifications/notification-service-api.js

```diff
@@ -0,0 +1,200 @@
+  NOTIFICATION_SERVICE_ACCOUNT,
+  NOTIFICATION_SERVICE_LOGIN,
+  NOTIFICATION_SERVICE_VERIFY,
+  NOTIFICATION_SERVICE_SUBSCRIPTIONS,
+  NOTIFICATION_SERVICE_TOKEN_KEY,
+  NOTIFICATION_SERVICE_EMAIL_KEY,
+  EXPIRED_TOKEN_MESSAGE,
+  ExpiredTokenError,
+  UnauthroizedError,
+} from './constants'
+import { getEthNetworkType } from '../../../local-settings'
+
+
+const isTokenExpired = response =>
+  response.statusCode === 401 && response.message === EXPIRED_TOKEN_MESSAGE
+
+const isUnauthorized = rawResponse => rawResponse.status === 401
+
+export const login = async ({ email, dao, network }) => {
+  try {
+    const rawResponse = await fetch(NOTIFICATION_SERVICE_LOGIN, {
+      method: 'POST',
+      headers: {
+        'Content-Type': 'application/json',
+      },
+      body: JSON.stringify({ email, dao, network }),
+    })
+    if (!rawResponse.ok) {
+      throw new Error(rawResponse.statusText)
+    }
+  } catch (e) {
+    console.error(e)
+    throw e
+  }
+}
+
+// Verify the short lived email token and fetch a long lived token
+export async function verifyEmailToken(shortLivedToken) {
+  try {
+    const rawResponse = await fetch(NOTIFICATION_SERVICE_VERIFY, {
+      method: 'POST',
+      headers: {
+        'Content-Type': 'application/json',
+        authorization: shortLivedToken,
+      },
+    })
+
+    if (!rawResponse.ok) {
+      // In case of errors the api will return json payload
+      const response = await rawResponse.json()
+
+      if (isTokenExpired(response))
+        throw new ExpiredTokenError(response.message)
+
+      if (isUnauthorized(rawResponse)) {
+        throw new UnauthroizedError(rawResponse.statusText)
+      }
+
+      throw new Error(rawResponse.statusText)
+    }
+
+    // Get the long lived token from header
+    return rawResponse.headers.get('authorization')
+  } catch (e) {
+    console.error(e.message)
+    throw e
+  }
+}
+
+/**
+ * Delete a user account
+ *
+ * @param {string} token long lived api token
+ * @returns {Number} 1 if the account was deleted successfully
+ */
+export async function deleteAccount(token) {
+  try {
+    const rawResponse = await fetch(NOTIFICATION_SERVICE_ACCOUNT, {
+      method: 'DELETE',
+      headers: {
+        'Content-Type': 'application/json',
+        authorization: token,
+      },
+    })
+
+    const response = await rawResponse.json()
+
+    if (!rawResponse.ok) {
+      // In case of errors the api will return json payload
+      if (isTokenExpired(response))
+        throw new ExpiredTokenError(response.message)
+
+      if (isUnauthorized(rawResponse)) {
+        throw new UnauthroizedError(rawResponse.statusText)
+      }
+
+      throw new Error(rawResponse.statusText)
+    }
+
+    localStorage.removeItem(NOTIFICATION_SERVICE_TOKEN_KEY)
+    localStorage.removeItem(NOTIFICATION_SERVICE_EMAIL_KEY)
+
+    return response
+  } catch (e) {
+    console.error(e.message)
+    throw e
+  }
+}
+
+export const getSubscriptions = async ({ token }) => {
```

Comment:
For consistency with `deleteAccount()`:

```suggestion
export const getSubscriptions = async (token) => {
```

Or we could change `deleteAccount()` to take in an object.